### PR TITLE
how-to: setup a shared cache (extracted from use cases)

### LINKED ISSUE #2487: guide: extract shared cache info from shared dev server case
> Split from https://github.com/iterative/dvc.org/issues/654#issuecomment-738557458

Per https://github.com/iterative/dvc.org/pull/2482#pullrequestreview-660486847:

> This ideally should be part of something else. Some ideas:
UG -> Data Management -> Shared Cache
[Or] in the How to, we can have - "How to avoid duplication of data in the team", "How to organize team storage for large raw datasets", etc, etc.

Rel https://github.com/iterative/dvc.org/issues/144#issuecomment-841960760

### PR DESCRIPTION
**Pull request by jorgeorpinel**
REPURPOSED to close #2487

---

Would close (no longer) https://github.com/iterative/dvc.org/issues/654#issuecomment-738557458
And (not yet) #820

- [ ] Consider comments from https://discuss.dvc.org/t/multiple-users-in-one-repo/746 (common file permission questions)
- [x] Work on or prioritize #2487 after this.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Just a quick draft so far @shcheklein . WDYT?

File: content/docs/user-guide/how-to/share-a-dvc-cache.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
-# Shared Development Server
+---
+title: 'How to Share a Cache Among Projects'
```
Comment (jorgeorpinel) [inline content/docs/user-guide/how-to/share-a-dvc-cache.md:2]: Note: Git thinks that I moved the UC to this HT directly since most of the content is similar so far (the steps starting in **Preparation**). In reality this is the new file and content/docs/use-cases/sharing-resources.md is the moved case (which also preserves some of the existing case's content).

File: content/docs/sidebar.json [hunk 2]
```diff
@@ -131,7 +131,8 @@
           "stop-tracking-data",
           "update-tracked-data",
           "add-deps-or-outs-to-a-stage",
-          "merge-conflicts"
+          "merge-conflicts",
+          "share-a-dvc-cache"
```
Comment (shcheklein) [inline content/docs/sidebar.json:135]: This ideally should be part of something else. Some ideas:

UG -> Data Management -> Shared Cache


here is in the How to, we can have - "How to avoid duplication of data in the team", "How to organize team storage for large raw datasets", etc, etc.

File: content/docs/use-cases/sharing-resources.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+$ git push
+```
+
+Your colleagues can [checkout](/doc/command-reference/checkout) the
+<abbr>project</abbr> data (from the shared <abbr>cache</abbr>), and have both
+`raw` and `clean` data files appear in their workspace without moving anything
+manually. After this, they could decide to continue building this
+[pipeline](/doc/command-reference/dag) and process the clean data:
+
+```dvc
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources.md:37]: there should be no CLI commands ... at least at the very beginning, especially so complicated

File: content/docs/use-cases/sharing-resources.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Sharing Resources
+
+DVC enables different ways to optimize resource utilization for scenarios where
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources.md:3]: let's for now try to start w/o mentioning DVC. It should be about the problem/use case in general first. 

e.g. Team -> lots of data -> run a lot of training -> manage data effificiently, distribute GPUs, etc.

File: content/docs/sidebar.json (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/sidebar.json:135]: > should be part of something else... UG -> Data Management -> Shared Cache

Agree . Didn't want to overcomplicate this PR. Should we go for a how-to for now and make an issue about the Data Mgmt section (would also lead to #520) instead?

File: content/docs/use-cases/sharing-resources.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources.md:37]: > should be no CLI commandsSure. I didn't spend much time in the content beyond the first few paragraphs yet for this draft.

Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources.md:37]: > should be no CLI commands

Sure. I didn't spend much time in the content beyond the first few paragraphs yet for this draft.

File: content/docs/sidebar.json (hunk 2 above)
Comment (shcheklein) [reply content/docs/sidebar.json:135]: I'm fine with that. Let's create a ticket though to address it later.

Comment (jorgeorpinel) [reply content/docs/sidebar.json:135]: > Should we go for a how-to

I meant top-level guide. I agree that the topic is a bit broad for a how.

File: content/docs/use-cases/sharing-resources.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources.md]: I updated the intro (and added a first section on storage). PTAL

File: content/docs/sidebar.json (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/sidebar.json:135]: Done: #2487

Comment (shcheklein) [reply content/docs/sidebar.json:135]: 👍

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use og special
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:3]: og - typo?

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+it's location, so nothing prevents you from having it in a
+[location shared](/doc/user-guide/how-to/share-a-dvc-cache) by multiple local
+copies of a <abbr>project</abbr>, or even by different projects altogether. This
+enables DVC's automatic de-duplication of data files across all projects.
+
+Additionally, optional [remote storage](/doc/command-reference/remote) e.g.
+Amazon S3 or Azure Blob Storage (managed separately) can be used by multiple
+projects to synchronize their caches (see `dvc push` and `dvc pull`).
+
+<!--
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:36]: remove this?

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+DVC projects support different ways to optimize resource utilization in order to
+minimize cost and complexity. This can make the difference, for example when:
+
+- Multiple users work on the same shared server, or there's a single computing
+  environment to run experiments.
+- GPU time gets distributed among people or processes for training machine
+  learning models.
+- There's a centralized data storage unit or cluster.
+
+## Shared Storage
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:19]: let's do the whole page w/o subsections first (and ideally w/o them at the end)

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+[versions of data](/doc/use-cases/versioning-data-and-model-files), results,
+etc. (think **Git for data**).
+
+The cache directory is fully customizable (see `dvc config cache`), including
+it's location, so nothing prevents you from having it in a
+[location shared](/doc/user-guide/how-to/share-a-dvc-cache) by multiple local
+copies of a <abbr>project</abbr>, or even by different projects altogether. This
+enables DVC's automatic de-duplication of data files across all projects.
+
+Additionally, optional [remote storage](/doc/command-reference/remote) e.g.
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:32]: Somewhere here we should get back to the initial problem we stated and explain how does this solution help. Also solution itself -we should jump a bit higher level and simplify it further.

File: content/docs/user-guide/how-to/share-a-dvc-cache.md [hunk 9]
```diff
@@ -77,44 +85,3 @@ If you're using Git, commit the changes to your project's config file (usually
 $ git add .dvc/config
 $ git commit -m "config external/shared DVC cache"
 ```
-
-## Examples
```
Comment (shcheklein) [inline content/docs/user-guide/how-to/share-a-dvc-cache.md:81]: why removing this?

**Review by shcheklein — CHANGES_REQUESTED** [review]
Good iteration of the use case - close to what we've been doing with other use cases. A bit of polishing is still required, but it's a good one indeed! We are close.

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 6 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md:36]: Yes, sorry it's still partly draft [WIP]. Will remove

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 7 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md:19]: OK

File: content/docs/user-guide/how-to/share-a-dvc-cache.md (hunk 9 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:81]: This isn't removed. Git thinks I moved the old use case into this new file. Explanation in https://github.com/iterative/dvc.org/pull/2482#pullrequestreview-660479451

Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:81]: I could try to undo everything, change the other file first in a commit, and then add this file in a separate commit so the diff is cleaner.

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 8 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: > get back to the initial problem we stated and explain how does this solution help
solution itself -we should jump a bit higher level

I added some clarifications both for local cache and remote storage. Not sure if that makes it higher level, I guess it's still a bit specific (referring to the 2 basic optimizations built into the data cache).

**Comment by shcheklein** [comment]
looks like there are some changes that are not related to this PR?

**Comment by jorgeorpinel** [comment]
Oh woops sorry, need to update the PR base... ⌛

**Comment by jorgeorpinel** [comment]
OK the first version of this story is complete! See https://dvc-org-cases-shareing--sizh7a.herokuapp.com/doc/use-cases/sharing-resources-efficiently (deploying rn ⌛)

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use of special
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:3]: files -> large amounts of data, datasets

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+processors, and minimize data transfers. This involves provisioning and managing
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:4]: rotate the use of special processors -> utilize available GPUs or share them ... or something like that

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+resources such as massive on-prem or cloud data stores and powerful servers,
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:5]: I would read this statement not as a problem ... but rather as a ... statement :)

Can we just skip it for simplicity ... or move `Minimizing ...` thing  here, closer and simplify this sentence

"... it might require clusters, some software to manage workloads, etc .."

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+which is expensive and time consuming.
+
+![](/img/resource-pool.png) _Fragile and disjointed pool of limited resources_
+
+Minimizing operational cost and complexity can make the difference, for example,
+when:
+
+- Multiple users work on the same shared server.
+- There's a centralized data storage unit or single-entry cluster.
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:14]: `or single-entry cluster.` - no sure I understand this term

Comment (shcheklein) [reply content/docs/use-cases/sharing-resources-efficiently.md:14]: are all these statement _or_ or _and_

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+- There's a single computing environment with access to the data needed to
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:15]: these two do not look different/overlap/clarify the previous two it seems?

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+  reproduce full [experiments](/doc/user-guide/experiment-management).
+- Distributing GPU time among people or processes for training machine learning
+  models
+
+In DVC, a built-in <abbr>caching</abbr> mechanism already provides individual
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:20]: people reading this might not realize I think that cloning DVC project means a Git repo

that they don't deal with some crazy infrastrucutre, etc ... we need to go bit higher I think here and may be in some precious paragraphs

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+[the same cache](/doc/user-guide/how-to/share-a-dvc-cache) directory in all of
+them so that they share a central data location, optimizing local (or network)
+storage.
+
+![](/img/shared-server.png) _One data store shared among people or projects_
+
+Additionally, [remote storage](/doc/command-reference/remote) (Amazon S3, SSH,
+Google Drive, etc.) can also be shared by multiple projects to synchronize their
+data caches (see `dvc push` and `dvc pull`). This can reduce service costs by
+consolidating the backups.
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:39]: backups - not the best term here. To some extent we are interested more in these optimizations when people store datasets there ... even for models - this is not about backups to my mind - it's a primary storage

Comment (shcheklein) [reply content/docs/use-cases/sharing-resources-efficiently.md:39]: though with shared cache scenario, it can be a backup when we specifically use the external cache that has everything in it as a primary storage, but it's not clear from the doc, and it is one of the possible scenarios

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+
+> 💡 Another way to centralize some or all of the data requirements of your
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:41]: this note is not very clear to be honest ...

File: content/docs/use-cases/sharing-resources-efficiently.md [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+> projects is to implement a [data registry](/doc/use-cases/data-registries)
+> pattern and leverage `dvc import`.
+
+Data-driven teams may also need a specialized execution environment (such as a
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-resources-efficiently.md:45]: not sure I follow this to be honest ... also, not clear if we need this? can we consolidate some important thoughts into the material above?

File: content/docs/command-reference/destroy.md [hunk 19]
```diff
@@ -16,12 +16,14 @@ usage: dvc destroy [-h] [-q | -v] [-f]
...
-cache, DVC will replace them with the latest versions of the actual files and
-directories first, so that your data is intact after destruction.
+set to an [external location][external-cache] (by default a local cache is
+located in `.dvc/cache`). If you have setup
+[symlinks](/doc/user-guide/large-dataset-optimization) (from cache to workspace)
+in your project, DVC will replace them with the latest versions of the actual
+files and directories first, so that your data is intact after destruction.
+
+[external-cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
```
Comment (jorgeorpinel) [inline content/docs/command-reference/destroy.md:26]: Unrelated note: what if we start extracting long links to this format? Especially if they have `#anchors`. It can help us separate those links to manage them better (rn it's hard to detect when they break).

> Stole the idea from @iesahin 🕵️

Comment (shcheklein) [reply content/docs/command-reference/destroy.md:26]: Need to check if the link checker supports this.

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 12 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: PTAL

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 13 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: Or. It's a list of example situations where "Minimizing cost and complexity can make the difference".

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 14 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: Not sure which 2 you were referring. But maybe the work "computing" was making it slightly confusing? Removed now.

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 15 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md:19]: OK. I removed details and made other small changes that help clarify for now. But I'll try to go even higher level... ⌛

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 16 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: Clarified (I think).

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 17 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: Hmmm. I mentioned "central" in the 2 previous paragraphs now. Does that help?

We're talking about ways that DVC can centralize data storage with cache and remote, so this tip about data registries as another alternative seemed relevant.

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 18 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md:43]: Right. The last few Ps def. need more work... ⌛

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 15 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: OK I rewrote this part again, PTAL.

File: content/docs/use-cases/sharing-resources-efficiently.md (hunk 18 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: OK I reworked this too.

Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-resources-efficiently.md]: I think the ideas are clearer now but the text is probably too long now...

File: content/docs/command-reference/destroy.md (hunk 19 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/destroy.md:26]: > Need to check if the link checker supports this.

I wasn't sure how to double check this 😢. Did it seem to work?

Comment (jorgeorpinel) [reply content/docs/command-reference/destroy.md:26]: From https://github.com/iterative/dvc.org/pull/2491#discussion_r645739606 it sounds like this isn't supported yet ): but we should probably open an issue about that instead of limiting our MD links?

**Comment by jorgeorpinel** [comment]
@shcheklein I've repurposed this to only move the shared cache info to the guide for now. The existing use case will remain untouched but no longer linked from everywhere. Please review content/docs/user-guide/how-to/share-a-dvc-cache.md when you have a chance. All other changes are links and sidebar.

File: content/docs/command-reference/gc.md [hunk 20]
```diff
@@ -81,10 +81,10 @@ The default remote is cleaned (see `dvc config core.remote`) unless the
   checkpoints).
 
 - `-p <paths>`, `--projects <paths>` - if a single remote or a single cache is
-  shared among different projects (e.g. a configuration like the one described
-  [here](/doc/use-cases/shared-development-server)), this option can be used to
-  specify a list of them (each project is a path) to keep data that is currently
-  referenced from them.
+  shared among different projects (e.g. as described
+  [here](/doc/use-cases/sharing-resources-efficiently)), this option can be used
```
Comment (shcheklein) [inline content/docs/command-reference/gc.md:85]: it's not part of this PR, right? should the use case one go first?

File: content/docs/use-cases/versioning-data-and-model-files/index.md [hunk 21]
```diff
@@ -62,10 +62,11 @@ Benefits of our approach include:
   DVC [optimizes](/doc/user-guide/large-dataset-optimization) storing and
   transferring large files.
 
-- **Collaboration**: Easily distribute your project development and share its
-  data [internally](/doc/use-cases/shared-development-server) and
-  [remotely](/doc/use-cases/sharing-data-and-model-files), or
-  [reuse](/doc/start/data-and-model-access) it in other places.
+- **Collaboration**: Easily distribute your project development and
+  [share limited resources](/doc/use-cases/sharing-resources-efficiently).
```
Comment (shcheklein) [inline content/docs/use-cases/versioning-data-and-model-files/index.md:66]: same - do we keep this link here?

File: content/docs/user-guide/how-to/share-a-dvc-cache.md [hunk 22]
```diff
@@ -0,0 +1,87 @@
+---
+title: 'How to Share a Cache Among Projects'
```
Comment (shcheklein) [inline content/docs/user-guide/how-to/share-a-dvc-cache.md:2]: do we remove the use case file as part of this PR?

File: content/docs/user-guide/how-to/share-a-dvc-cache.md [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+
+1. You have distributed copies of a DVC repository in a single shared server
+   with multiple users. A shared cache is necessary to avoid duplicating the
+   project's data on the single local storage available to all.
+2. Your team works with multiple projects in environments with limited storage,
+   which share a large storage unit. Everyone needs to use the shared drive
+   anyway, and combining the cache locations will also prevent data duplication
+   (across projects).
+
+## Preparation
```
Comment (shcheklein) [inline content/docs/user-guide/how-to/share-a-dvc-cache.md:20]: Are there any changes to the things below, compared to the existing doc?

File: content/docs/user-guide/managing-external-data.md [hunk 24]
```diff
@@ -65,7 +65,7 @@ location, e.g. `s3://mybucket/cache` or `ssh://user@example.com/cache` (see the
 examples below).
 
 > ⚠️ An external cache could be
-> [shared](/doc/use-cases/shared-development-server) among copies of a DVC
+> [shared](/doc/use-cases/sharing-resources-efficiently) among copies of a DVC
```
Comment (shcheklein) [inline content/docs/user-guide/managing-external-data.md:68]: same here - and check other place please

File: content/docs/user-guide/how-to/share-a-dvc-cache.md [hunk 25]
```diff
@@ -0,0 +1,87 @@
+---
+title: 'How to Share a Cache Among Projects'
+description: >-
+  Setup a single cache for different projects or distributed copies of a same
+  project.
+---
+
+# How to Share a DVC Cache
+
+There are 2 main reasons to setup a shared <abbr>DVC cache</abbr>:
```
Comment (shcheklein) [inline content/docs/user-guide/how-to/share-a-dvc-cache.md:10]: both items are pretty hard to read to be honest

so, we have a few cases people would use this:

- one large machine (e.g. with multiple GPUs) and one storage on it and people do the same or multiple projects and we want to avoid duplication (no copies) and save time (no copy)
- one large NAS (<examples here>) and people attach it to their machines - again no copies - fast, doesn't take extra space, fits even if there are no space (ability to work with really large data)


what else am I missing? can we make description more explicit/simpler?

File: content/docs/command-reference/gc.md (hunk 20 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/gc.md:85]: Ah yes thanks. Fixing...

File: content/docs/use-cases/versioning-data-and-model-files/index.md (hunk 21 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/versioning-data-and-model-files/index.md:66]: Fixing...

File: content/docs/command-reference/gc.md (hunk 20 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/gc.md]: Fixed.

> should the use case one go first?

Hmm the order isn't that important I think. Since this one is ready, why not go for it. This way we don't need to worry about where to put the shared cache info. later when the shared dev server use case changes completely.

File: content/docs/user-guide/how-to/share-a-dvc-cache.md (hunk 22 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:2]: Not doing that yet. It's basically the same content, yes so we could. But I thought there were enough changes in this PR. Up to you.

Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:2]: (being removed in #2587 anyway)

Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:2]: I'd err on the side of keeping it until the new use case is ready though. Also removing it may require a redirect (yet another file to change).

File: content/docs/user-guide/how-to/share-a-dvc-cache.md (hunk 23 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:20]: Nope. The Example is gone though, as it doesn't make sense in a how-to.

File: content/docs/user-guide/managing-external-data.md (hunk 24 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/managing-external-data.md]: Oops. All fixed now, my bad. That was sloppy 😓

File: content/docs/user-guide/how-to/share-a-dvc-cache.md (hunk 25 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md]: Agreed. Updated, PTAL.

**Review by shcheklein — CHANGES_REQUESTED** [review]
There are two images - do we use those?

Do we need to setup a redirect?

File: content/docs/user-guide/how-to/share-a-dvc-cache.md (hunk 22 above)
Comment (shcheklein) [reply content/docs/user-guide/how-to/share-a-dvc-cache.md:2]: Okay, got it.

**Comment by jorgeorpinel** [comment]
🤦 removed the images.

No doc is removed yet so I see no need for a redirect. I did change all the relevant links to the how-to for now.

#2587 will have the redirect though.

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Sanity checked https://dvc-org-cases-shareing--2pfth4.herokuapp.com/doc/user-guide/how-to/share-a-dvc-cache 🚀
